server/flow.go

```diff
@@ -264,8 +271,7 @@ func (fm *FlowManager) stepCSversionGreaterthan9() flow.Step {
 				fmt.Sprintf("   - **Redirect URL**: `%s`\n", fm.GetRedirectURL()) +
 				"   - **Application Permissions**: `Admin`\n" +
 				"   Select **Continue**.\n" +
-				"5. Copy the `clientID` and `clientSecret` from **Settings**, and paste them into the modal in Mattermost which can be opened by using the `/confluence config add` slash command.\n" +
-				"6. In Mattermost, use the `/confluence connect` slash command to connect your Mattermost account with your Confluence account\n",
+				"5. Copy the `clientID` and `clientSecret` from **Settings**\n",
```

Comment:
Why this change?

Reply:
we do not have a config add command, secret and id is used in the next step of the flow